amplify/src/main/java/com/github/stkent/amplify/tracking/checks/CooldownDaysCheck.java

```diff
@@ -33,7 +33,7 @@ public CooldownDaysCheck(final long cooldownPeriodDays) {
 
     @Override
     public boolean shouldBlockFeedbackPrompt(@NonNull final Long cachedEventValue, @NonNull final Context applicationContext) {
-        return (System.currentTimeMillis() - cachedEventValue) < TimeUnit.DAYS.toMillis(cooldownPeriodDays);
+        return cachedEventValue == Long.MAX_VALUE || (System.currentTimeMillis() - cachedEventValue) < TimeUnit.DAYS.toMillis(cooldownPeriodDays);
```

Comment:
This will block if the event has never happened which is not what we want! (We do want that for the warmup event though).